cmd/auto-updater/windows.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+// Implements support for running the storage node as a Windows Service.
```

Comment:
This comment needs an update: "storage node" --> "auto-updater for storage nodes".

Reply:
ah missed that one, thanks